.github/workflows/ci-wordpress.yaml

```diff
@@ -0,0 +1,85 @@
+name: CI-Wordpress
+
+# ----------------------------------------------------------------------------------------------------------
+# CI Pipeline for deploy wordpress to shared hosting by ansistrano ansible role
+#
+# Original Author: Rostislav Udaltsov
+# Created: 09/19/2023
+# ----------------------------------------------------------------------------------------------------------
+
+on:
+  workflow_call:
+    inputs:
+      # Build configuration
+      production_host:
```

Comment:
what will happen if the client decide to introduce staging/test envs?
always build your solutions expecting this to happen. So adjust your implementation accordingly.

Reply:
see how we did it here
https://github.com/saritasa-nest/saritasa-github-actions/blob/918cbb86ac7abe0b3cff06f79e965c6b776812a5/.github/workflows/ci-ios.yaml#L12

Reply:
done

Reply:
In this implementation, the introduction of an additional variable such as environment does not give anything, we always know unequivocally for which environment we are deploying and based on this we specify the path to the folder on the server. The only possible use that I see for this variable is to check the compliance of the branch from which the deployment is made to the environment, for example 

- `dev` from `develop` branch 
or
- `production` from `main` branch

Reply:
But you place this solution in "reusable" actions of the `saritasa-github-actions` - and such actions should be dev-agnostic.

Reply:
cleaning, from transferred resources, deleting irrelevant branches